### ISSUE TITLE
OverflowError: line number table is too long - when importing visual from latests psychopy

### ISSUE THREAD

**Issue by matthiasdold** [psychopy/psychopy#5091]
Hi all,

I am encountering the following `OverflowError` when installing `psychopy` from pip.

Steps to reconstruct
```bash
python --version    # Python 3.10.6

python -m venv testenv
source testenv/bin/activate.fish

pip install -U pip
pip install attrdict3 wxpython psychopy

python -c "import psychopy; print(psychopy.__version__)"                                                                                                                                                                             
2022.2.3
```
Then running a simple import fails like this

```bash
 python -c "from psychopy import visual"      
                                                                                                                                                                                        
pygame 2.1.2 (SDL 2.0.16, Python 3.10.6)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/md/workspace/python/venvs/testvenv/lib/python3.10/site-packages/psychopy/visual/__init__.py", line 38, in <module>
    from .textbox2.textbox2 import TextBox2
  File "/home/md/workspace/python/venvs/testvenv/lib/python3.10/site-packages/psychopy/visual/textbox2/__init__.py", line 4, in <module>
    from .textbox2 import TextBox2, allFonts
  File "/home/md/workspace/python/venvs/testvenv/lib/python3.10/site-packages/psychopy/visual/textbox2/textbox2.py", line 32, in <module>
    from psychopy.tools.linebreak import get_breakable_points, break_units
  File "/home/md/workspace/python/venvs/testvenv/lib/python3.10/site-packages/psychopy/tools/linebreak.py", line 12, in <module>
    from psychopy.tools.linebreak_class import linebreak_class
OverflowError: line number table is too long
```
Searching for this Overflow error leads to reports during conversion of large float or storing a pandas data frame as `hdf5`.

The problem is that linebreak_class is actually a single dictionary with >300k lines and abou 4.5Mb in size:
```bash
wc -l testenv/lib/python3.10/site-packages/psychopy/tools/linebreak_class.py                                                                                                         
333244 testenv/lib/python3.10/site-packages/psychopy/tools/linebreak_class.py

python -c "from psychopy.tools.linebreak_class import linebreak_class"                                                                                                     
Traceback (most recent call last):
  File "<string>", line 1, in <module>
OverflowError: line number table is too long
```

A super crude workaround which I am currently using is this
```bash
sed -i -z 's/\n//g' testenv/lib/python3.10/site-packages/psychopy/tools/linebreak_class.py
```

Any clue on how to fix this correctly?

**Comment by smathot** [comment]
Same issue on Ubuntu 22.04 with PsychoPy 2022.1.1. Oddly, of two systems with (as far as I can tell) identical software, only one of the two suffered from this error.

**Comment by cantordust** [comment]
Same issue encountered on OpenSuSE Tumbleweed.

**Comment by mdcutone** [comment]
Linux compatibility isn't great right now, lots of stuff broke with Python 3.10. Presently working on it.

**Comment by TEParsons** [comment]
I believe #5185 fixes this :)

**Comment by delale** [comment]
> Linux compatibility isn't great right now, lots of stuff broke with Python 3.10. Presently working on it.

Same on OSX Sonora; fixed by downgrading to Python 3.9.18
